src/main/webapp/event-list.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Event List</title>
+<link rel="stylesheet" href="/css/main.css">
+<link rel="stylesheet" href="/css/user-page.css">
+
+<script>
+    
+  // Fetch messages and add them to the page.
+  function fetchEvents(){
+    const url = "/event-list";
+    fetch(url).then((response) => {
+      return response.json();
+    }).then((events) => {
+      const eventContainer = document.getElementById('event-container');
+      if(events.length == 0){
+       eventContainer.innerHTML = '<p>There are no events at this time. \
+       Please check again later</p>';
+      }
+      else{
+       eventContainer.innerHTML = '';  
+      }
+      events.forEach((event) => {  
+       const eventDiv = buildEventDiv(event);
+       eventContainer.appendChild(eventDiv);
+      });
+    });
+  }
+  
+  function buildEventDiv(event){
+    //For the event name
+   const eventNameDiv = document.createElement('div');
+   eventNameDiv.classList.add("left-align");
+   eventNameDiv.appendChild(document.createTextNode(event.eventName));
+
+   //Organizers name
+   const organizerNamesDiv = document.createElement('div');
+   organizerNamesDiv.classList.add("left-align");
+   organizerNamesDiv.appendChild(document.createTextNode(event.organizerNames));
+   
+   /**Will need some way to nicely format the date/time of the event later on*/
+   const dateDiv = document.createElement('div');
+   dateDiv.classList.add('right-align');
+   dateDiv.appendChild(document.createTextNode(new Date(event.eventDate)));
```

Comment:
The Date(String)` constructor is actually deprecated since it's not very obvious how a String should be converted into a Date (see https://docs.oracle.com/javase/8/docs/api/java/util/Date.html#Date-java.lang.String-).  But more than that, right now we're just storing `eventDate` as a String, so there's nothing guaranteeing that it will be in the specific format expected by `new Date()`.  So for now let's just show whatever is in `eventDate` directly, and in the future we can see about replacing `eventDate` with a timestamp or something that we can safely render using a date/time library.